.github/workflows/deployment.yml

```diff
@@ -26,3 +26,29 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        id: login-dockerhub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+        
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v4
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: :${{ steps.meta.outputs.tags }}
```

Comment:
Are we missing the name of the image or am I missing something? Shouldn't this be `tags: bledy/timezone-converter:${{ steps.meta.outputs.tags }}`

Reply:
Thanks for pointing that out @ibLeDy  
I have added it.